Classes/Controller/FormController.php

```diff
@@ -360,7 +360,7 @@ public function initializeObject()
             $logger = ObjectUtility::getLogger(__CLASS__);
             $logger->info('Powermail settings', $this->settings);
         }
-        $this->signalDispatch(__CLASS__, __FUNCTION__ . 'Settings', [$this]);
+        $this->signalDispatch(__CLASS__, __FUNCTION__ . 'Settings', [&$this->settings, $this]);
```

Comment:
Please use $this->settings as second parameter. So there is less breaking changes on existing installations.